src/summary/history/programs/programs-card.component.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import style from "./conditions-card-style.css";
+import dayjs from "dayjs";
+import SummaryCard from "../../cards/summary-card.component";
+import SummaryCardRow from "../../cards/summary-card-row.component";
+import SummaryCardRowContent from "../../cards/summary-card-row-content.component";
+import { match } from "react-router";
+import { performPatientProgramsSearch } from "./programs.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import HorizontalLabelValue from "../../cards/horizontal-label-value.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import SummaryCardFooter from "../../cards/summary-card-footer.component";
+import { Trans, useTranslation } from "react-i18next";
+
+export default function Programs(props: ProgramsCardProps) {
+  const [patientPrograms, setPatientPrograms] = React.useState(null);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const subscription = performPatientProgramsSearch(patientUuid).subscribe(
```

Comment:
nit: I think this function could be named simpler, perhaps `getPatientPrograms` or `fetchPatientPrograms`

Reply:
Done. Changed to fetchPatientPrograms